static/service-worker.js

```diff
@@ -22,24 +21,25 @@ self.addEventListener('activate', (event) => {
 self.addEventListener('fetch', function(event) {
   const accept = event.request.headers.get('Accept');
   if (accept === 'application/json' || event.request.method !== 'GET' ||
-      event.request.mode !== 'no-cors') {
+      !['no-cors', 'navigate'].includes(event.request.mode)) {
```

Comment:
Does this PR still intend to always return cached 'index.html'？
If the browser requests `index.html`, `event.request.mode` would be `'navigate'`. This PR seems to cache 'index.html' and return cache if cached.

Reply:
No, it will return whatever the initial request was for. This PR changes what is actually cached, such that HTML, CSS, etc. are also cached, allowing us to serve up the entire page when the user is offline.

Reply:
Ah, I see what you mean. Yes, that is a problem during development, but the entire cache is invalidated with every single build during development, so that's not specific to the index.html case. In a release build, the cache version will remain constant until an update.

For testing purposes, I made the cache version constant by changing `VERSION` in the webpack config.

Reply:
> so that's not specific to the index.html case.

Maybe yes. Currently, almost build artifacts have version prefix(change per build) and it referenced from `index.html`, so that with refreshed `index.html` build artifacts autoimatically would be refreshed althought old version are chached.

However, it's OK. I was just wondering.
I understand that sometime we should force-reload to browse new UI when update or rebuild the Gateway.